manifests/install.yaml

```
@@ -17240,7 +17253,7 @@ spec:
           - key: tls.key
             path: tls.key
           - key: ca.crt
-            path: ca.crt
+            path: ca.crtrgocd-notifications-controller
```

I think this is an accidental addition.
```suggestion
            path: ca.crt
```

---

Oh, thank you! Completely my bad, fixing it right now.